### DISCUSSION TITLE
unable to use numbering or get the output as base64 sting by using patchDocument function

### DISCUSSION THREAD

**Discussion by MahmoudY3c** [dolanmiu/docx#2389] [General]
in docx js pather it a very nice solution but when modify a file it's acting like `Document` class but it's doesn't have the features of `Document` class like convert result to blob, buffer, base64 string + use numbering and always return the result as arraybuffer which even not readable for document class is there's a way to get the result of patcher to something readable for `Document` class or to add the features of document to the patcher

**Comment by dolanmiu** [comment]
>  blob, buffer, base64 string + use numbering and always return the result as arraybuffer

This is now possible with: https://github.com/dolanmiu/docx/pull/2497
